docs/Course Wiki/Other Course Offerings/STAT400.md

```diff
@@ -0,0 +1,64 @@
+---
+tags:
+- Probability
+---
+# STAT400
+
+STAT400 (Statistics and Probability I) is a four credit hour statistics and probability course intended as an introduction for math-oriented students.
+
+## Content Covered
+
+- Discrete and continuous random variables
+- Basic probability distributions
+- Moment generating functions
+- Normal distribution
+- Discrete and continuous bivariate distributions
+- Conditional distributions
+- Covariance and correlation coefficients
+- Maximum Likelihood Estimation, Method of Moments estimation, Mean Squared Error
+- Confidence intervals
+- Hypothesis testing
+- Chi-square tests
+- Basic R usage
+
+STAT400 assumes very little knowledge of statistics but does require understaning of calculus. The introductory topics of statistics are covered quite quickly, and the course moves on to more advanced topics such as hypothesis testing and confidence intervals.
+
+## Prerequisites
+
+- [MATH231](../MATH%20Course%20Offerings/MATH231.md)
+- Concurrent enrollment in [MATH241](../MATH%20Course%20Offerings/MATH241.md) is required
```

Comment:
Change to 
`- [MATH241](../MATH%20Course%20Offerings/MATH241.md) (corequisite)`